docs/Absolute.md

```diff
@@ -1,12 +1,16 @@
 
 # Absolute
 
-Use `<Absolute />` component to control position.
-This component is an extension of the `<Box />` component.
+Use `<Absolute />` component to control position.  Use with a `<Relative>` parent/ancestor.
+
+This component is an extension of the [`<Box />`](https://pricelinelabs.github.io/design-system/Box) component.   It inherits `<Box />` props.
```

Comment:
Tiny code format nit: if your Markdown line is > 80 characters, you can break to next line to start with a space to continue writing, but with "_cleaner_" code format:
```
It is really long ...,
 now it continues...
```

Reply:
FWIW, with GFM the space at the beginning of the line isn't needed

Reply:
Updated.  Thanks for the tip!